code/c/uppercaser/uppercaser.c

```diff
@@ -12,7 +12,6 @@
 #include <stdio.h>
 
 /* function declarations */
-void print_number_of_args(int num_args);
```

Comment:
I don't think this should be removed. But I think it's what was supposed to be called in **line 20**, which seems to be incorrectly named :    ```  // printNumberOfArgs(argc);```

It should be called like this ```print_number_of_args(argc);``` instead.
You can submit these changes :) (keep it commented though, see the line above)